### PR TITLE
Show add-on runs or events after dispatching

### PR DESCRIPTION
**Pull request by eyeseast**

### CODE REVIEW

**Comment by duckduckgrayduck** [comment]
Looking at this, after dispatch it sends us to the Add-Ons history page where it shows the run queueing and running. Just to confirm, we don't want to replicate the Add-On banner from old DocumentCloud?

**Comment by eyeseast** [comment]
@duckduckgrayduck Look at #735. That's our hub for ongoing processes. But my thinking here is that if you're dispatching an add-on, I'm going to show you the thing you just ran.

**Review by allanlasser — COMMENTED** [review]
Some small design notes focused on providing clearer user feedback.

File: src/lib/components/forms/AddOnDispatch.svelte [hunk 1]
```diff
@@ -89,9 +92,32 @@
       { event: "disabled", selection: null },
     );
   }
+
+  /** @type {import('@sveltejs/kit').SubmitFunction} */
+  function onSubmit({ submitter }) {
+    submitter.disabled = true;
```
[the remark this answers is not in the document]
Comment (eyeseast) [reply src/lib/components/forms/AddOnDispatch.svelte:99]: Changed the button text to a spinning icon while the add-on is running. (It's fast, so dial your network down to 3G to see it.) I will do another PR with error feedback.

**Review by allanlasser — APPROVED** [review]

**Pull request merged**